### PR TITLE
Bump minimum required PHP version to 7.0 and minimum required WP version to 6.3

### LINKED ISSUE #169: Fix failing test on WP 6.0
### Is your enhancement related to a problem? Please describe.

In #166 a test was skipped that consistently failed on WP 6.0.

We need to figure out whether the test is broken or the actual implementation, and fix it.

### Designs

_No response_

### Describe alternatives you've considered

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### LINKED ISSUE #222: Bump minimum required PHP version to 7.0 and minimum required WP version to 6.3
### Is your enhancement related to a problem? Please describe.

As WordPress bumps the [minimum required PHP version to 7.0](https://core.trac.wordpress.org/ticket/57345), we should do the same for plugins.

### Designs

_No response_

### Describe alternatives you've considered

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by mukeshpanchal27**
Closes #222 
Closes #169

### Checklist:

- [x] I agree to follow this project's [**Code of Conduct**](https://github.com/10up/.github/blob/trunk/CODE_OF_CONDUCT.md).

### CODE REVIEW

File: .github/workflows/php-test.yml [hunk 1]
```diff
@@ -59,9 +59,9 @@ jobs:
         wordpress: [ 'latest' ]
         include:
           - php: '8.0'
-            wordpress: '6.0'
-          - php: '8.0'
-            wordpress: '6.1'
+            wordpress: '6.3'
+          - php: '8.1'
+            wordpress: '6.3'
```
Comment (swissspidy) [inline .github/workflows/php-test.yml]: Right now, these two combinations are already covered by 8.1+latest and 8.0+latest above

We can remove them now and re-add once 6.4 is latest

**Review by swissspidy — COMMENTED** [review]
One nit otherwise looks good

File: .github/workflows/php-test.yml (hunk 1 above)
Comment (mukeshpanchal27) [reply .github/workflows/php-test.yml]: Thanks. Removed in https://github.com/WordPress/plugin-check/pull/292/commits/6ce9148503555293d46c4874c59d2c5049683e60

**Review by swissspidy — APPROVED** [review]

**Review by felixarntz — APPROVED** [review]
Thanks @mukeshpanchal27, LGTM!

**Pull request merged**